test/trufflescripts/add_token_pair.js

```diff
@@ -90,16 +93,17 @@ module.exports = async () => {
   console.log('FundingUSD == ', startingETH * ethUSDPrice)
   console.log('Auction Index BEFORE == ', (await dx.getAuctionIndex.call(sellToken.address, buyToken.address)).toNumber())
 
-  const funds = sell === 'eth' ?
-    [web3.toWei(10, 'ether'), 0, 2, 1] :
-    [0, web3.toWei(10, 'ether'), 1, 2]
+  const funds = sell === 'eth'
+    ? [web3.toWei(10, 'ether'), 0, 2, 1]
+    : [0, web3.toWei(10, 'ether'), 1, 2]
 
   await dx.addTokenPair(
     sellToken.address,                            // -----> SellToken Address
     buyToken.address,                           // -----> BuyToken Address
     ...funds,                                    // -----> sellFund, buyFund, closingPriceNum, closingPriceDen
     { from: account },
   )
+  await mineCurrentBlock()
   console.log('Auction Index AFTER == ', (await dx.getAuctionIndex.call(sellToken.address, buyToken.address)).toNumber())
-  return mineCurrentBlock()
+  process.exit()
```

Comment:
does it hang otherwise?

Reply:
i was testing some stuff, not important. tbh the trufflescripts stuff isn't good, you should just ignore it